mcs/class/corlib/Test/System/GetInterfaceTest.cs

```diff
@@ -0,0 +1,38 @@
+// GetInterfaceTest.cs - NUnit Test Cases for https://github.com/mono/mono/issues/6579
+//
+// Konstantin Khitrykh (konh@yandex.ru)
+//
+// (C) Konstantin Khitrykh
+// 
+
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace MonoTests.System {
+	[TestFixture]
+	public class GetInterfaceTest {
```

Comment:
Not sure it's worth to add a new test file, just add the method to existing TypeTest.cs